master/types.html

```diff
@@ -1614,8 +1584,8 @@ <h2 id="DOMInterfacesForReflectingSVGAttributes">DOM interfaces for reflecting S
 <ul>
   <li>If the type of the reflecting IDL attribute is a
   <a href="http://heycam.github.io/webidl/#dfn-primitive-type">primitive type</a> (such as
-  <b>long</b>, as used by <a href="types.html#__svg__SVGElement__tabIndex">tabIndex</a>
-  on <a>SVGElement</a>) or <b>DOMString</b> (as used by
+  <b>long</b>, as used by <a href="types.html#__svg__SVGNameList__numberOfItems">numberOfItems</a>
+  on <a>SVGNumberList</a>) or <b>DOMString</b> (as used by
```

Comment:
This seems to be an additional change that doesn't seem to belong to the mixin change. I am unsure if this general reference to a `SVGNumberList` item is correct (also why not simply referencing `SVGNumberList` instead of giving an obscure example?)

Also, SVGNumberList uses `unsigned long` and not `long`.

Reply:
I was removing the tabIndex reference as it is no longer defined in the spec and thus the link wouldn't work. I originally went with baseVal as that is a long in the same section but it was confusing with the section below it that talks about SVGAnimatedString. I guess I could find another example. (SVGNumberList links to that generic SVGNameList as many share the same definition, so it was probably a bad example to pick)